__init__.py

```diff
@@ -1,3 +1,3 @@
 __name__ = "fun"
 __version__ = "0.0.1"
-__all__ = ()
+__all__ = ("random")
```

Comment:
This has to be `("random", )`, otherwise it is considered to be a string, not tuple.